test/check-machines-create

```diff
@@ -1060,8 +1060,6 @@ class TestMachinesCreate(VirtualMachinesCase):
             return self
 
         def assert_pixels(self, subtag=None):
-            # focus away from any input field, to avoid the text cursor in screenshots
-            self.browser.focus("#create-vm-dialog .btn-cancel")
```

Comment:
That is correct, but will again change the pixel tests. The update commit was earlier than this change, but let's see if you included it there already.

Reply:
No changes were necessary. :-)